guides/v2.3/cloud/live/go-live-checklist.md

```diff
@@ -1,49 +1,33 @@
 ---
 group: cloud-guide
-subgroup: 165_live
 title: Go live checklist
-menu_title: Go live checklist
-menu_order: 10
-menu_node:
 functional_areas:
   - Cloud
   - Testing
 ---
 
-Prior to entering your ticket to go live and switching the DNS, you should complete this checklist and all tests for your deployed site/store. Deploy your `master` branch to the Production environment.
+## Pre-Launch Checklist {#checklist}
```

Comment:
A heading right after a heading is weird.

Reply:
Agreed-- I removed it.